### PR TITLE
Toolkit Migration For Remaining Samples

### PR DESCRIPTION
**Pull request by Mohammed-MSFT**
Included Samples
1. App Complete Auth - Nodejs
2. Tab-graph-toolkit -Nodejs
3. Bot-release-management -Nodejs

### CODE REVIEW

**Review by formulahendry — CHANGES_REQUESTED** [review]
Please update readme as well.

**Comment by Mohammed-MSFT** [comment]
> Please update readme as well.

File: samples/app-sso/nodejs/appManifest/manifest.json
Comment (formulahendry) [inline samples/app-sso/nodejs/appManifest/manifest.json:1]: This file content is not changed?

File: samples/app-sso/nodejs/teamsapp.local.yml [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+  # Teams Developer Portal.
+  # Will use the app id in manifest file to determine which Teams app to update.
+  - uses: teamsApp/update
+    with:
+      # Relative path to teamsfx folder. This is the path for built zip file.
+      appPackagePath: ./appManifest/build/appManifest.${{TEAMSFX_ENV}}.zip
+
+deploy:
+  # Run npm command
+  - uses: cli/runNpmCommand
```
Comment (formulahendry) [inline samples/app-sso/nodejs/teamsapp.local.yml:80]: do you need install for 'client' folder?

File: samples/app-sso/nodejs/teamsapp.local.yml
Comment (formulahendry) [inline samples/app-sso/nodejs/teamsapp.local.yml:1]: have you tested it form a clean environment, Will it work?

File: samples/bot-release-management/nodejs/aad.manifest.json [hunk 2]
```diff
@@ -0,0 +1,123 @@
+{
```
Comment (formulahendry) [inline samples/bot-release-management/nodejs/aad.manifest.json:1]: Do this project use SSO?

File: samples/bot-release-management/nodejs/teamsapp.yml [hunk 3]
```diff
@@ -0,0 +1,4 @@
+version: 1.0.0
+
+environmentFolderPath: ./env
+projectId: fb82297a-277f-4c38-a0a6-4cfd479c7127
```
Comment (formulahendry) [inline samples/bot-release-management/nodejs/teamsapp.yml:4]: remove projectId

File: samples/bot-release-management/nodejs/aad.manifest.json (hunk 2 above)
Comment (Mohammed-MSFT) [reply samples/bot-release-management/nodejs/aad.manifest.json:1]: No, but this samples required application permissions
image

File: samples/tab-graph-toolkit/nodejs/teamsapp.local.yml [hunk 4]
```diff
@@ -0,0 +1,85 @@
+# yaml-language-server: $schema=https://aka.ms/teams-toolkit/1.0.0/yaml.schema.json
```
Comment (formulahendry) [inline samples/tab-graph-toolkit/nodejs/teamsapp.local.yml:1]: this project need an AAD App. Will this work?

Comment (Mohammed-MSFT) [reply samples/tab-graph-toolkit/nodejs/teamsapp.local.yml:1]: Yes tested this application
image

Comment (Mohammed-MSFT) [reply samples/tab-graph-toolkit/nodejs/teamsapp.local.yml:1]: Yes, tested this sample.
image

File: samples/bot-release-management/nodejs/teamsapp.yml (hunk 3 above)
Comment (Mohammed-MSFT) [reply samples/bot-release-management/nodejs/teamsapp.yml:4]: Fixed

File: samples/app-sso/nodejs/appManifest/manifest.json
Comment (Mohammed-MSFT) [reply samples/app-sso/nodejs/appManifest/manifest.json:1]: fixed

File: samples/app-sso/nodejs/teamsapp.local.yml (hunk 1 above)
Comment (Mohammed-MSFT) [reply samples/app-sso/nodejs/teamsapp.local.yml:80]: yes, this sample has both client and server.

File: samples/app-sso/nodejs/teamsapp.local.yml
Comment (Mohammed-MSFT) [reply samples/app-sso/nodejs/teamsapp.local.yml:1]: Yes sample is working.
image

File: samples/app-sso/nodejs/teamsapp.local.yml (hunk 1 above)
Comment (formulahendry) [reply samples/app-sso/nodejs/teamsapp.local.yml:80]: I mean: you need to do 'npm install' for 'client' folder as well. But, in this file, you did not do it.

**Pull request closed without merging**